src/installer/corehost/cli/bundle/info.h

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __INFO_H_
+#define __INFO_H_
+
+#include "error_codes.h"
+#include "header.h"
+
+// bundle::info supports:
+// * API for identification of a single-file app bundle, and
+// * Minimal probing and mapping functionality only for the app.runtimeconfig.json and app.deps.json files.
+// bundle::info is used by HostFxr to read the above config files.
+
+namespace bundle
+{
+    struct info_t
+    {
+        struct config_t
+        {
+            config_t() {}
```

Comment:
This doesn't initialize `m_location`, and `config_t::matches()` will dereference that pointer.  When `info_t::m_deps_json`and `info_t::m_runtimeconfig_json` are constructed, they use this constructor here.